config/environments/production.rb

```diff
@@ -128,4 +128,7 @@
   config.x.hses.client_id = ENV["HSES_AUTH_CLIENT_ID"]
   config.x.hses.client_secret = ENV["HSES_AUTH_CLIENT_SECRET"]
   config.x.bypass_auth = false
+
+  # API configuration
+  config.x.use_real_api_data = true
```

Comment:
[thought] probably too late to be thinking this thought, but it might be nice to have these vars/logic so that it doesn't need to do anything special to use real data, so we didn't have to set extra vars in production.
like... `config.x.bypass_real_api` (you can feel free to ignore this thought, as it probably complicates current work, and may not actually simplify anything)

Reply:
> so that it doesn't need to do anything special to use real data

I'm not sure I'm following. In production I just hardcoded it to always use the real data.